test/mox_test.exs

```diff
@@ -495,7 +540,7 @@ defmodule MoxTest do
 
       {:ok, child_pid} =
         Task.start_link(fn ->
```

Comment:
Instead of using `assert_default_raise`, we should rather change to a `spawn_link` or call `Process.delete(:"$callers")`.